packages/components/src/index.js

```diff
@@ -39,7 +39,16 @@ import ObjectViewer from './ObjectViewer';
 import Progress from './Progress';
 import Skeleton from './Skeleton';
 import SidePanel from './SidePanel';
-import { Table, TableCell, TableHeaderCell } from './Table';
+import {
+	DataAccessorWithSorterAndFilter,
+	FilterFactory,
+	Sorter,
+	SortOrder,
+	Table,
+	TableCell,
+	TableHeaderCell,
+	TableSortHeader,
+} from './Table';
```

Comment:
components index is supposed to only contains components, please remove all the things that are not components